news/2020-08-03-mappers-report-july.md

```diff
@@ -0,0 +1,183 @@
+
+### BN Departures
+
+- **osu!:** [Peter](https://osu.ppy.sh/users/8623835), [Kaitjuh](https://osu.ppy.sh/users/2225327), [Icekalt](https://osu.ppy.sh/users/5410645) & [Rizen](https://osu.ppy.sh/users/3056732)
+- **osu!taiko:** [Volta](https://osu.ppy.sh/users/4154071)
+- **osu!catch:** [JBHyperion](https://osu.ppy.sh/users/4879508)
+
+### NAT Additions 
+
+A new member in the Nomination Assessment has risen! Please welcome [Scotty](https://osu.ppy.sh/users/11085809) to the team! After around one and a half year as a Beatmap Nominator, he will now help the current osu!mania NAT with the Beatmap Nominator evaluations and make sure that the constant flow of new members recently will not break up. 
+Welcome aboard Scotty!
+
+## <a id="mappers-showcase"></a>Mappers' Showcase
+
+Another month, another Mappers' Showcase! However, the term "mapper" is maybe not the right one this time. Since the showcase has covered all game modes at least once, we thought we'd make something special this time. We would like to show you the work of storyboarders!
+As storyboarding can apply to all game modes, we thought it'd be the perfect moment to showcase some outstanding storyboarders this month!
+
+This month got covered by [Walavouchey](https://osu.ppy.sh/users/5773079) with two excellent storyboarder picks!
+
+### ohm002
+
+Having made quite a few storyboards in only the short span of, well, this year, [ohm002](https://osu.ppy.sh/users/4468239) has been building up their repertoire trying out a lot of different ideas. While being newer in the scene, they have managed some rather quick improvement that aspiring storyboarders might want to take note of!
+
+In their first ranked storyboard for [Halozy - Paranoid Lost](https://osu.ppy.sh/beatmapsets/1180913#osu/2462333), ohm002 goes big and bold with lots of contrast between sections of the song. While not the cleanest in the technicalities, it sure gives a shocking impression and a unique playing experience.
+
+In contrast, their recent work for [HAG - Rokugatsu no Himitsu Kichi](https://osu.ppy.sh/beatmapsets/1094362#osu/2287170) shows a rather clean and concise style with less particle effects. They demonstrate how a simple style can look good when you get the fundamentals right. There's clearly a lot of potential here and we look forward to seeing their future work!
+
+### PantyDev
+
+As a long time storyboarder, [PantyDev](https://osu.ppy.sh/users/5420543) has an impressive variety in their portfolio. With storyboards ranging from simple and clean to complex and ambitious, they might just have the experience to compete with the best.
+
+A unique storyboard to start with would definitely be their visualization for [BoxPlot - Tramontane](https://osu.ppy.sh/beatmapsets/1016398#osu/2127254), basically turning the whole background into a swaying curtain. Certainly a neat concept that not many storyboards use, though it may feel a bit dizzying.
```

Comment:
```suggestion
A unique storyboard to start with would definitely be their visualization for [BoxPlot - Tramontane](https://osu.ppy.sh/beatmapsets/1016398#osu/2127254), basically turning the whole background into a swaying curtain. Certainly, a neat concept that not many storyboards use, though it may feel a bit dizzying.
```